.github/workflows/weekly_tests.yaml

```diff
@@ -15,6 +15,9 @@ jobs:
       fail-fast: false
       matrix:
         include:
+          - repo: bootstack-actions
+            workflow_file_name: pull-request.yaml
+            branch: main
```

Comment:
For bootstack-actions, we use `pull-request.yaml` as a reusable workflow file in other repos and is not something that needs to be run weekly.